modules/core/src/action.ts

```diff
@@ -21,28 +21,58 @@ const hasOwnProperty = <P extends string | number>(
 }
 
 /**
- * Action
+ * Data format to send messages among components
  * @class
  */
 export abstract class Action<V, T = AType> {
   public abstract readonly type: T
   public abstract readonly value: V
+  /**
+   * Create new Action
+   * @param type Address of the message
+   * @param value Content of the message
+   */
   static of<T extends AType, V>(type: T, value: V): Action<V, T> {
     return Action.isNil(value) ? value : new VAction(type, value)
   }
-
+  /**
+   * Return true if parameter passed is an action
+   * @param a
+   */
   static isAction(a: unknown): a is Action<unknown> {
     return a instanceof Action
   }
 
+  /**
+   * Return true if parameter passed is a Nil action
+   * @param a
+   */
   static isNil(a: unknown): a is Nil {
     return a instanceof Nil
   }
-
+  /**
+   * Creates new nil action
```

Comment:
- Some explanatino of what a `Nil` action is?
- Also, capitalize the `nil` word, since it's an actual entity.